saleor/product/filters.py

```diff
@@ -1,147 +0,0 @@
-from collections import OrderedDict, defaultdict
-from itertools import chain
-from typing import Dict, Iterable
-
-from django.db.models import Exists, OuterRef, Q, QuerySet
-from django.forms import CheckboxSelectMultiple
-from django_filters import MultipleChoiceFilter, OrderingFilter, RangeFilter
-
-from ..core.filters import SortedFilterSet
```

Comment:
We can also remove the `saleor/core/filters.py` file which defined the base class for filtering in the old dashboard, it is not used anymore.

Reply:
I will remove that